.yarn/patches/lavamoat-browserify-npm-15.5.0-ce0bbe5e4c.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/src/index.js b/src/index.js
+index 6943a606618d0841ad68eb0927f33b1267ab6ff2..f12aec0376e779415036c2b7c22e3932502b6817 100644
+--- a/src/index.js
++++ b/src/index.js
+@@ -8,6 +8,7 @@ const createLavaPack = require('@lavamoat/lavapack')
+ const { createSesWorkaroundsTransform } = require('./sesTransforms')
+ const { loadCanonicalNameMap } = require('@lavamoat/aa')
+ const browserResolve = require('browser-resolve')
++const resolve = require('resolve');
+ 
+ 
+ // these are the reccomended arguments for lavaMoat to work well with browserify
+@@ -71,7 +72,7 @@ function plugin (browserify, pluginOpts) {
+       browserify.pipeline.get('emit-deps').push(createModuleInspectorSpy({
+         policyOverride,
+         // no builtins in the browser (yet!)
+-        isBuiltin: () => false,
++        isBuiltin: (name) => browserify._options.node && resolve.isCore(name),
```

Comment:
Ok, this might address my worries about figuring out if we should import an actual builtin or a package.
But it doesn't seem to determine what happens in  loadModuleData, since the check is happening there independently (and should probably be the same as here)

Reply:
AFAIK this check is only for building the policy file. 

In the LavaMoat runtime the `loadModuleData` call happens before the policy itself can be retrieved for a given package. So it would require some restructuring to do things differently: https://github.com/LavaMoat/LavaMoat/blob/main/packages/lavapack/src/runtime.js#L11244-L11259